.markdownlint-cli2.yaml

```diff
@@ -0,0 +1,20 @@
+config:
+  line-length:
+    line_length: 120
```

Comment:
Is there an exception for code blocks and xref lines?

Reply:
normally urls should not affect it - but i only migrated this from the old to the new file